src/spatial/models/report_datafetcher.py

```
@@ -0,0 +1,152 @@
+import requests
+import torch 
```

Ruff

`torch` imported but unused

Remove unused import: `torch`

(F401)